cwl/src/test/scala/wdl4s/cwl/ParseBigThreeSpec.scala

```diff
@@ -9,6 +9,8 @@ class ParseBigThreeSpec extends FlatSpec with Matchers {
   val namespace = "cwl"
 
   it should "parse 1st tool" in {
+    //This id:message is just boilerplate because our object requires an id
+    // and we don't support the map[] declaration yet
```

Comment:
Map declarations are indeed supported by SALAD.  deleting this comment as it no longer applies